src/main/java/twilightforest/item/ItemTFMoonwormQueen.java

```diff
@@ -29,123 +32,92 @@ protected ItemTFMoonwormQueen() {
 	}
 
 	@Override
-	public ActionResult<ItemStack> onItemRightClick(ItemStack par1ItemStack, World world, EntityPlayer player, EnumHand hand) {
-		if (par1ItemStack.getItemDamage() < this.getMaxDamage()) 
+	public ActionResult<ItemStack> onItemRightClick(ItemStack stack, World world, EntityPlayer player, EnumHand hand) {
+		if (stack.getItemDamage() < getMaxDamage(stack))
 		{
 			player.setActiveHand(hand);
 		}
 		else 
 		{
 			player.resetActiveHand();
 		}
-		return ActionResult.newResult(EnumActionResult.SUCCESS, par1ItemStack);
+		return ActionResult.newResult(EnumActionResult.SUCCESS, stack);
 	}
 
 	@Override
-	public EnumActionResult onItemUse(ItemStack par1ItemStack, EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing side, float hitX, float hitY, float hitZ)
+	public EnumActionResult onItemUse(ItemStack stack, EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing side, float hitX, float hitY, float hitZ)
```

Comment:
`onItemUse` is mostly copy-pasted from ancient `ItemBlock.onItemUse`. can you modernize it based on the new ItemBlock and add a `// [VanillaCopy]` comment stating that it's heavily based on vanilla code?